scripts/merge-config.escript

```diff
@@ -12,7 +12,7 @@
 
 main(_) ->
     {ok, BaseConf} = file:read_file("apps/emqx_conf/etc/emqx_conf.conf"),
-    Cfgs = get_all_cfgs("apps/"),
+    Cfgs = get_all_cfgs("apps/") ++ get_all_cfgs("lib-ee/"),
```

Comment:
does this merge ee configs to ce release ?

Reply:
This part is updated now.